components/dropdown/basic/src/index.scss

```diff
@@ -134,18 +137,26 @@
       @include reset-list;
 
       &Item {
-        &:not(:first-child) {
-          margin-top: $m-v;
+        @include media-breakpoint-down(s) {
+          margin-bottom: $m-v-large;
```

Comment:
This is an old variable. $m-m ?

Reply:
yes it is, I will change this variable and other old variables from this component :+1: